flamingchickens1540/inflatedchickens
# Admin Dashboard

## Description
The admin dashboard now displays all scouts and robots in their respective queues; clicking on a scout or robot will remove them from the queue.

## Testing
1. Open home and admin page
2. Join the queue
3. Remove yourself from the queue on the admin side
4. Rejoin and remove yourself from the scout side
5. Rejoin and queue a match, five robots should appear in the queue
